### PR TITLE
feat(app): liquid updates in protocol details and setup for odd & desktop

### PR DESCRIPTION
**Pull request by smb2268**
fix PLAT-438, PLAT-440, PLAT-441, PLAT-439, PLAT-442, PLAT-443, PLAT-444

# Overview
Updates to liquid details on ODD and desktop protocol details and setup

## Test Plan and Hands on Testing
Look at a protocol with liquids and examine liquid details on protocol details and protocol setup

Screen Shot 2024-08-02 at 3 38 09 PM
Screen Shot 2024-08-02 at 3 31 23 PM
Screen Shot 2024-08-02 at 3 30 57 PM
Screen Shot 2024-08-02 at 4 39 15 PM
Screen Shot 2024-08-02 at 3 58 41 PM

## Changelog
1. Remove volume column from protocol details view for ODD and desktop 
2. Add column titles to run setup view for liquid name and total volume  for ODD and desktop 
3. Change list items in setup view to show volume per well instead of total volume in that labware  for ODD and desktop 
4. Show DeckInfoLabel instead of plain slot name for labware location on desktop for Flex only

## Review requests
Look at screenshots for matching designs, look over code

## Risk assessment
Low

### CODE REVIEW

File: app/src/organisms/Devices/ProtocolRun/SetupLiquids/SetupLiquidsList.tsx [hunk 1]
```diff
@@ -19,28 +20,26 @@ import {
   LiquidIcon,
   SIZE_AUTO,
   SPACING,
-  LegacyStyledText,
-  TYPOGRAPHY,
   StyledText,
+  JUSTIFY_SPACE_BETWEEN,
```
Comment (ncdiehl11) [inline app/src/organisms/Devices/ProtocolRun/SetupLiquids/SetupLiquidsList.tsx]: nit, move up to alphabetize

**Comment by ecormany** [comment]
We need to round the individual well volume numbers to avoid floating point precision errors. Nearest tenth of a µL should be good. 
image

File: app/src/organisms/ProtocolDetails/ProtocolLiquidsDetails.tsx [hunk 2]
```diff
@@ -54,13 +53,27 @@ export const ProtocolLiquidsDetails = (
...
+                      marginX={SPACING.spacing16}
+                    >
+                      {liquid.displayName}
+                    </StyledText>
+                    <StyledText
+                      desktopStyle="bodyDefaultRegular"
+                      color={COLORS.grey60}
+                      marginX={SPACING.spacing16}
+                    >
+                      {liquid.description != null ? liquid.description : null}
```
Comment (ncdiehl11) [inline app/src/organisms/ProtocolDetails/ProtocolLiquidsDetails.tsx]: Looks like liquids require a description to pass analysis, so this should always be non-null
```{liquid.description}```

**Review by ncdiehl11 — APPROVED** [review]
Looks good to me after addressing Ed's rounding comment

**Pull request merged**